dashboards/health.erb

```diff
@@ -44,15 +44,18 @@ $(function() {
       <li data-row="1" data-col="12" data-sizex="1" data-sizey="1">
         <div data-id="offender-case-notes-prod" data-view="ServerStatusSquares" data-title="Offender Case Notes - Prod"></div>
       </li>
-      <li data-row="1" data-col="13" data-sizex="1" data-sizey="1">
+      <li data-row="1" data-col="12" data-sizex="1" data-sizey="1">
```

Comment:
merge gone a bit haywire - I'd leave this alone otherwise there will be two 12 items